docs/versioned_docs/version-3.0.0/introduction/support.md

```diff
@@ -0,0 +1,37 @@
+---
+id: support
+title: OpenEBS Community
+sidebar_label: Community
+---
+
+---
```

Comment:
Is this line required ?

Reply:
Might have come from `support docs from staging`, will format it.